frontend/src/server/Room.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+var app = require("express")();
+var http = require("http").Server(app);
+var io = require("socket.io")(http);
+
+const port = 3000;
+
+app.get("/", function(req, res) {
+  res.sendFile(__dirname + "/index.html");
+});
+
+io.on("connection", function(socket) {
+  console.log("a user connected");
```

Comment:
I would strongly suggest adding a bit more information about the user if possible, potentially like their IP address to be able to differentiate between players.

It's also the case that we're going to have to deal with connecting, disconnecting, and reconnecting. We're going to need to handle cookies or some other way of allowing players to return to an existing game state.